DoSomethingArchive/legacy-website
# removes #prefix and #suffix helper text

#### What's this PR do?

Removes `#prefix` and `#suffix` (title and description helper texts) from file upload since it was showing up on the user view and not just the form like intended. 
#### How should this be manually tested?

Visit a campaign page to the Reportback Image section and make sure `The image must be at least 480 by 480 pixels. The site will crop the image on its own, so if possible submit a square image with the subject in the center of the crop.` helper text doesn't appear. 
#### Any background context you want to provide?

`#title` and `#description` were not appearing as helper text for the the file upload section of `/admin/users/rb/add` form so I used `#prefix` and `#suffix` as alternatives. `#theme_wrappers' => array(),` seems to be the line that breaks `#title` and `#description`. 
#### What are the relevant tickets?

Removes helper text from user-facing view issue in #6184. Will open separate PR to close out issue.
